contracts/Vault.vy

```diff
@@ -250,7 +244,7 @@ def withdraw(listing: Listing, active_rental: Rental, sender: address) -> uint25
 
     # clear vault and set listing to zero to uninitialize
     self.unclaimed_rewards = 0
-    self.listing = empty(bytes32)
+    self.listing = empty_listing_hash
```

Comment:
this has a different meaning
`self.listing == empty(bytes32)` means uninitialized vault
`self.listing == empty_listing_hash` means initialized with `empty(Listing)`